src/olympia/devhub/tests/test_views_edit.py

```diff
@@ -38,56 +38,67 @@ def get_section_url(addon, section, edit=False):
 
 
 @override_settings(MEDIA_ROOT=None)  # Make it overridable.
-class TestEdit(TestCase):
+class EditTest(TestCase):
```

Comment:
When it's just a base class I like to indicate that in the name, so that when you pick up the code months later you know that the fact that that test class isn't run directly by the test runner was intentional.